src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -727,6 +751,7 @@ def get_devices(_sys_block_path='/sys/block', _dev_path='/dev', _mapper_path='/d
         metadata['human_readable_size'] = human_readable_size(float(size) * 512)
         metadata['size'] = float(size) * 512
         metadata['path'] = diskname
+        metadata['exclusive_access'] = is_writable_raw_device(metadata['path'])
```

Comment:
See, this is the inconsistency that the code will hit if not following a pattern for naming. The key says `exclusive_access` but it is calling a function that is named `is_writable_raw_device`, so which one is it?

Reply:
Fixed the semantic. Thx.

Reply:
Fixed the semantic. Thx.